airbyte-webapp/src/packages/cloud/views/auth/components/PersonQuoteCover/PersonQuoteCover.module.scss

```diff
@@ -0,0 +1,84 @@
+@use "../../../../../../scss/colors";
+@use "../../../../../../scss/fonts";
+
+.image {
+  position: relative;
+  height: 100vh;
+  background-image: url(../../../../../../../public/person-photo.png);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+
+  .overlay {
+    position: absolute;
+    height: 100%;
+    width: 100%;
+    background-image: linear-gradient(
+      45deg,
+      colors.$orange-400 10%,
+      rgba(colors.$orange-400, 0.8) 30%,
+      rgba(colors.$blue-400, 0.8) 80%,
+      colors.$blue-400 100%
+    );
+    background-repeat: no-repeat;
+    background-size: cover;
+    backdrop-filter: grayscale(1);
```

Comment:
This effect does not work in Firefox. I see the photo colorized. Maybe it's best to grayscale the image?
https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter#browser_compatibility